File: CS/Form1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.XtraReports.UI;

namespace dxSample {
    public partial class Form1 : DevExpress.XtraEditors.XtraForm {
        public Form1() {
            InitializeComponent();
        }

        private void simpleButton1_Click(object sender, EventArgs e) {
            XtraReport2 report = new XtraReport2();
            report.CreateDocument(false);
            new ReportPrintTool(report).ShowPreviewDialog();
        }
    }
}

File: VB/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports DevExpress.XtraReports.UI

Namespace dxSample
    Partial Public Class Form1
        Inherits DevExpress.XtraEditors.XtraForm

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub simpleButton1_Click(ByVal sender As Object, ByVal e As EventArgs)
            Dim report As New XtraReport2()
            report.CreateDocument(False)
            CType(New ReportPrintTool(report), ReportPrintTool).ShowPreviewDialog()
        End Sub
    End Class
End Namespace
